foundations/installations/installations.md

```diff
@@ -261,14 +256,106 @@ If you encounter an error requesting you to disable **Intel RST** while attempti
+
+Hyper-V is required to get WSL to work however, so in order to use it you'll first have to re enable Hyper-V, which may interfere with existing installations.
+
+- Search for Control Panel in your applications, and open it
+- Select **Programs**
+- Select **Turn Windows Features on or off** (under the Programs and Features header)
+- Select **Hyper-V**, toggle the checkbox to checked, and click **OK**
+- When prompted, restart your computer
+
+#### Step 1.2: Installing WSL
+
+- Open PowerShell in administrator mode by searching for it in your applications, right clicking, and then selecting run as administrator
+- Enter the following command
+
+~~~powershell
+wsl --install
+~~~
+
+- After a few minutes you'll be prompted to enter a new username. This username should be lowercase, but can otherwise be whatever suits you. You'll also need to enter a new password.
+
+#### Step 1.3.1: Install Windows Terminal (Windows 10 only)
+
+Windows Terminal is an terminal application that lets you more easily customize and run terminals, as well as supporting multiple tabs which can each run their own different terminals.
+
+- Follow [Microsoft's guide](https://learn.microsoft.com/en-us/windows/terminal/install) to install Windows Terminal.
+
+#### Step 1.3.2: Setting WSL as default (Optional)
+
+Unless you are regularly using other terminals on your computer we recommend that you set WSL as the default terminal program when you open Windows Terminal. You can follow the [section on setting a default profile](https://learn.microsoft.com/en-us/windows/terminal/install#set-your-default-terminal-application) in the guide above to set it to Ubuntu.
+
+### Step 2 Opening WSL
+
+On Windows there are two primary ways to open WSL: either via the terminal or via the app icon.
+
+- If you set Windows Terminal to open a Ubuntu terminal by default you can start a new WSL session by simply opening the terminal app.
+- If you search for Ubuntu in the application search bar you should see a application titled Ubuntu; simply open it to start a new terminal session.
+
+<div class="lesson-note lesson-note--warning">
+When opening WSL ensure that you do not see /mnt/c at the start of the line. /mnt/c is where your Windows installation lives when working within WSL, and messing around there can have unintended consequences
+</div>
+
+### Step 3: WSL and systemd
+
+`systemd` is a system and service manager for Linux that you'll use much later in the curriculum. While it doesn't work on WSL by default, you can enable it.
+
+- Open a new WSL terminal window.
+- Enter the following command:
+
+~~~bash
+sudo nano /etc/wsl.conf
+~~~
+
+- When prompted for a password enter the password you set earlier. You may notice that while entering your password you don't see any visual feedback. This is an intentional security measure, and nothing to worry about.
+- You'll be shown `nano`, a text editor for Linux. Enter the following two lines
+
+~~~
+[boot]
+systemd=true
+~~~
```

Comment:
This was already there for me for some reason

Reply:
Hmmmm, wasn't there for me on W11 having never touched the file; will add a note that if the text is already there you can disregard the rest of the instructions

Reply:
can't find this section anymore

Reply:
See above

Reply:
I dumped systemd for now as it was causing some people (including me) to have broken VSCode installations. Instructions on how to use it via WSL might still be useful, but I think thats better suited for a follow up issue